addons/website_event/controllers/main.py

```diff
@@ -1,24 +1,21 @@
 # -*- coding: utf-8 -*-
 
 import time
-import werkzeug.urls
+import werkzeug
 from datetime import datetime, timedelta
 from dateutil.relativedelta import relativedelta
 
-from openerp import http
-from openerp import tools, SUPERUSER_ID
+from openerp import http, _
 from openerp.addons.website.models.website import slug
 from openerp.http import request
-from openerp.tools.translate import _
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT as DF, DEFAULT_SERVER_DATETIME_FORMAT as DTF
```

Comment:
use fields.datetime.to_string() and fields.datetime.from_string() instead of DEFAULT_SERVER_DATE_FORMAT